### PR TITLE
commentscard

### PR DESCRIPTION
**Pull request by Nicolakirk**

### CODE REVIEW

File: src/components/CommentCard.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+    <h4>Created by {comment.author}</h4>
+    <p>{comment.body}</p>
+    <p>votes = {comment.votes}</p>
+    <p></p>
+  </li>
+)
+
+}
+
```
Comment (popatre) [inline src/components/CommentCard.jsx]: Lovely display component

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    const[isLoading, setIsLoading] =useState(true);
+    const[commentsList, setCommentsList]= useState([]);
+    const params = useParams();
+    const articleId = params.article_id;
+
+    useEffect(() => {
+        setIsLoading(true);
+        fetchCommentsbyArticleId(articleId).then((comments) => {
+          setCommentsList(comments);
+          console.log(comments);
```
Comment (popatre) [inline src/components/CommentList.jsx]: remove log

File: src/utils/api.js [hunk 3]
```diff
@@ -19,4 +19,11 @@ export const fetchArticles =  ()=>{
              
             return response.data.article
         })
+    }
+
+    export const fetchCommentsbyArticleId = (articleId)=> {
+        return newsApi.get(`/articles/${articleId}/comments`).then ((response) =>{
+            console.log(response.data.comments);
```
Comment (popatre) [inline src/utils/api.js]: remove log

File: src/components/CommentList.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        setIsLoading(false);
+        });
+      }, []);
+    
+    return (
+      <div>
+        <h2>  Here are all the comments about article {articleId}</h2>
+        <ul className="commentsboxes">
+            {commentsList.map((comment)=> {
```
Comment (popatre) [inline src/components/CommentList.jsx:35]: Could you add some message for when theres no comments?

File: src/components/CommentList.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        setIsLoading(true);
+        fetchCommentsbyArticleId(articleId).then((comments) => {
+          setCommentsList(comments);
+          console.log(comments);
+        setIsLoading(false);
+        });
+      }, []);
+    
+    return (
+      <div>
```
Comment (popatre) [inline src/components/CommentList.jsx:31]: Could you use the loading state to give some feedback to the user that the comment are loading?

**Review by popatre — CHANGES_REQUESTED** [review]
Lovely stuff - just a few things to add - see comments 👍

**Comment by NRMSMITH** [comment]
- [ ] you've declared isLoading in your CommentList but you haven't actually use it anywhere.
- [ ] check if there are any comments for the article - if there aren't, a message should appear for the user

**Comment by NRMSMITH** [comment]
Please have a think now if you want your comments to render on a separate page. It's absolutely fine if you want to do so and I'm happy for you to merge, but it's generally not how most users expect an app to work.
👍 Good to merge once you've considered that.

**Pull request merged**